### PR TITLE
fix: adjust EIP-1271 transaction service links

### PR DESCRIPTION
**Pull request by iamacook**
This updates the transaction service links from staging to productions.

### CODE REVIEW

File: learn/web/eip-1271.md [hunk 1]
```diff
@@ -13,16 +13,14 @@ It is possible to sign [EIP-191](https://eips.ethereum.org/EIPS/eip-712) complia
 ### Enabling off-chain signing
 
 Before off-chain signing it was and still is possible to sign messages on-chain. Multiple dApps exist in production which rely on this as the default behavior.
-Therefore, to enable off-chain signing it is necessary to send a custom RPC call `safe_setSettings` and pass in 
-the following to signal that the dApp wants to use off-chain signing:
+Therefore, to enable off-chain signing it is necessary to send a custom RPC call `safe_setSettings` and pass in the following to signal that the dApp wants to use off-chain signing:
```
Comment (iamacook) [inline learn/web/eip-1271.md:16]: My linter changed this.

File: learn/web/eip-1271.md [hunk 2]
```diff
@@ -138,7 +135,7 @@ const getSafeMessageHash = async (
 };
 ```
 
-Now we can query the state of the message from the tx-service's endpoint for messages: https://safe-transaction-goerli.staging.5afe.dev/api/messages/<SafeMsgHash>
+Now we can query the state of the message from the network-specific transaction service endpoint for messages: https://safe-transaction-<Network>.safe.global/api/messages/<SafeMsgHash>
```
Comment (ademidun) [inline learn/web/eip-1271.md:138]:
```suggestion
Now we can query the state of the message from the network-specific transaction service endpoint for messages: `https://safe-transaction-<Network>.safe.global/api/v1/messages/<SafeMsgHash>`.

For example here is an example response from Goerli: `https://safe-transaction-goerli.safe.global/api/v1/messages/`.

For other network endpoints, see [Available Services](/learn/safe-core-api/available-services.md).

```

Comment (ademidun) [reply learn/web/eip-1271.md:138]: Do you have a message hash URL we can use as an example here? Preferably Goerli but any example is fine.

**Review by ademidun — CHANGES_REQUESTED** [review]

File: learn/web/eip-1271.md (hunk 2 above)
Comment (iamacook) [reply learn/web/eip-1271.md]: I just created an [off-chain signature](https://safe-transaction-goerli.safe.global/api/v1/messages/0x7f4032fd13f27c5fce4694a1a6b79f65c68656da4eee4347a414f5bcec915b39/) specifically for the docs and added your suggestions.

File: learn/web/eip-1271.md [hunk 3]
```diff
@@ -138,7 +135,11 @@ const getSafeMessageHash = async (
 };
 ```
 
-Now we can query the state of the message from the tx-service's endpoint for messages: https://safe-transaction-goerli.staging.5afe.dev/api/messages/<SafeMsgHash>
+Now we can query the state of the message from the network-specific transaction service endpoint for messages: `https://safe-transaction-<Network>.safe.global/api/messages/<SafeMsgHash>`.
```
Comment (ademidun) [inline learn/web/eip-1271.md:138]:
```suggestion
Now we can query the state of the message from the network-specific transaction service endpoint for messages: `https://safe-transaction-<Network>.safe.global/api/v1/messages/<SafeMsgHash>`.
```

**Review by ademidun — APPROVED** [review]
Looks great! Thanks Aaron!

File: learn/web/eip-1271.md [hunk 4]
```diff
@@ -147,7 +148,7 @@ const fetchMessage = async (
   safeMessageHash: string
 ): Promise<TransactionServiceSafeMessage | undefined> => {
   const safeMessage = await fetch(
-    `https://safe-transaction-goerli.staging.5afe.dev/api/v1/messages/${safeMessageHash}`,
+    `https://safe-transaction-goerli.safe.global/api/messages/${safeMessageHash}`,
```
Comment (ademidun) [inline learn/web/eip-1271.md:151]:
```suggestion
    `https://safe-transaction-goerli.safe.global/api/v1/messages/${safeMessageHash}`,
```

**Pull request merged**